tests/unit/cn/mbset_test.c

```diff
@@ -122,7 +70,11 @@ fini(struct mtf_test_info *mtf)
 static int
 pre(struct mtf_test_info *mtf)
 {
-    mock_set();
+    mapi_inject(mapi_idx_mpool_mblock_delete, 0);
+    mapi_inject(mapi_idx_mblk_madvise, 0);
+    MOCK_SET_FN(mblk_desc, mblk_mmap, mocked_mblk_mmap);
+    MOCK_SET_FN(mblk_desc, mblk_munmap, mocked_mblk_munmap);
```

Comment:
Pro tip:  Use `MOCK_SET_FN` instead of `MOCK_SET` and avoid the need to have function names starting with `_`.

Reply:
Could be a good idea to just remove MOCK_SET all together then at some point